src/main/java/org/spongepowered/common/mixin/core/world/MixinWorld.java

```diff
@@ -239,7 +239,7 @@
     @Shadow public abstract void onEntityRemoved(net.minecraft.entity.Entity entityIn);
     @Shadow public abstract void updateEntity(net.minecraft.entity.Entity ent);
     @Shadow public abstract boolean isBlockLoaded(BlockPos pos);
-    @Shadow public abstract void markChunkDirty(BlockPos pos, net.minecraft.tileentity.TileEntity unusedTileEntity);
+    @Shadow public void markChunkDirty(BlockPos pos, net.minecraft.tileentity.TileEntity unusedTileEntity){};
```

Comment:
This change isn't necessary - unless the method is private, shadow methods should be marked abstract (even if the parent isn't)

Reply:
Maybe it's because I didn't have a body then, thanks.

Reply:
@dualspiral What do I need to do to fix it then? As far as I'm aware it is necessary, as MixinWorldServer needs to be able to call the super method if I want the code to compile. Since it's shadowed it's just a placeholder anyway right?

Reply:
Yeah @ryantheleach is correct, the call to `super` requires the parent method to not be abstract.

Reply:
Validated it in debugging by making hoppers pass items back and forth, then used evaluate expression to set the local to null.

It correctly ran the vanilla method.

Reply:
Ah, my mistake!

Reply:
I suggest throwing an `AbstractMethodError` here (to be consistent with shadowed static methods)

Reply:
@liach Is there a reason for that? it takes up more room, so was wondering if there was ever a usecase where it could be called. Also why AbstractMethodError?

Reply:
There isn't a use case. Shadowed methods that aren't existing will throw an exception at runtime for mixin application anyways.